### PR TITLE
Added missing services in the documentation

### PR DESCRIPTION
**Pull request by symphony-soufiane**
### Description
Added missing services in the documentation

### CODE REVIEW

File: docs/user.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  public static final Long USER_ID = 123456789L;
+
+  public static void main(String[] args) throws Exception {
+    // Create BDK entry point
+    final SymphonyBdk bdk = new SymphonyBdk(loadFromClasspath("/config.yaml"));
+
+    // Get user details
+    UserService users = bdk.users();
+    V2UserDetail userDetail = users.getUserDetail(USER_ID);
+    log.info("User details: " + userDetail);
```
Comment (symphony-youri) [inline docs/user.md:51]: where is the log coming from in the example?

Comment (symphony-soufiane) [reply docs/user.md:52]: I updated the examples with Slf4j annotation.

File: docs/disclaimer.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Disclaimer Service
+
+The Disclaimer Service is a component at the service layer of the BDK which aims to cover the Connections part of the [REST API documentation](https://developers.symphony.com/restapi/reference).
```
Comment (yinan-symphony) [inline docs/disclaimer.md:3]: _which aims to cover the **Connections** part_,   looks like it should be **Disclaimers**

File: docs/health.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+The Health Service is a component at the service layer of the BDK which covers the Health Service part of the [REST API documentation](https://developers.symphony.com/restapi/reference).
+More precisely:
+* [Health check](https://developers.symphony.com/restapi/reference/health-check-v3)
+* [Health check extended](https://developers.symphony.com/restapi/reference/health-check-extended-v3)
+* [Agent info](https://developers.symphony.com/restapi/reference/agent-info-v1)
+
+
+## How to use
+The central component for the Health Service is the `UserService` class, it exposes the service APIs endpoints mentioned above.
+The service is accessible from the`SymphonyBdk` object by calling the `users()` method:
```
Comment (yinan-symphony) [inline docs/health.md:11]: _Health Service is the `**UserService**` class_,   should it be **HealthService** ?  and **_health()_** for the next line maybe

File: docs/presence.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Connection Service
```
Comment (yinan-symphony) [inline docs/presence.md:1]: i think you mean "Presence Service"

File: docs/disclaimer.md (hunk 2 above)
Comment (yinan-symphony) [reply docs/disclaimer.md]: @symphony-soufiane did you change something regarding my comment?

**Review by yinan-symphony — APPROVED** [review]
LGTM

**Review by symphony-youri — APPROVED** [review]

**Pull request merged**